zingolib/src/shard/fetch_full_transaction.rs

```diff
@@ -414,19 +413,18 @@ impl TransactionContext {
         // Check if any of the nullifiers generated in this transaction are ours. We only need this for unconfirmed transactions,
         // because for transactions in the block, we will check the nullifiers from the blockdata
         if pending {
-            let unspent_nullifiers =
-            <<D as DomainWalletExt>
-              ::WalletNote as zingo_traits::ReceivedNoteAndMetadata>
-                ::Nullifier::get_nullifiers_of_unspent_notes_from_transaction_set(
-                &*self.transaction_metadata_set.read().await,
-            );
+            let unspent_nullifiers = self
+                .transaction_metadata_set
+                .read()
+                .await
+                .get_nullifier_value_txid_outputindex_of_unspent_notes::<D>();
```

Comment:
Should this be:   `get_nullifier_values_and_txid_outputindexes_of_unspent_notes`?